src/services/expenses.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+class Expenses {
+  constructor() {
+    this.data = [];
+    this.idCounter = 0;
+  }
+
+  init() {
```

Comment:
Having and `init` method together with identical constructor is redundant. You need to reset your data for tests, though, so you can call it reset. (same for user service)
```suggestion
  reset() {
```